ui/packages/consul-ui/app/components/certificate/README.mdx

```diff
@@ -0,0 +1,31 @@
+# Certificate
+
+```hbs preview-template
+<Certificate
+  @item="-----BEGIN CERTIFICATE-----
+  MIIH/TCCBeWgAwIBAgIQaBYE3/M08XHYCnNVmcFBcjANBgkqhkiG9w0BAQsFADBy
+  MQswCQYDVQQGEwJVUzEOMAwGA1UECAwFVGV4YXMxEDAOBgNVBAcMB0hvdXN0b24x
+  ETAPBgNVBAoMCFNTTCBDb3JwMS4wLAYDVQQDDCVTU0wuY29tIEVWIFNTTCBJbnRl
+  -----END CERTIFICATE-----"
+  @name="certificate-name"
+/>
+```
+
+### Arguments
+
+| Argument | Type | Default | Description |
+| --- | --- | --- | --- |
+| `item` | `String` | | The certificate as a string to be displayed and copied |
+| `name` | `String` | | The 'Name' of the certificate to be displayed and copied. Mainly used for giving feedback to the user. |
+
+This component has the following:
+- a copy button
+- a visibility button to show and hide certificate
+- a hidden and visible display of the certificate
```

Comment:
Ah I previewed this and I can see what you were saying about the `ul`s - the bullets don't show as we have them 'turned off' by default on the UI (the docs currently take all the styles from the UI). You can overwrite this in `debug.scss` if you want so the bullets show up just in the docs pages.

Reply:
I think it's fine. When I read these files I read them raw. The bullets do show in GitHub preview so it's 💯